build/rofl/artifacts.go

```diff
@@ -1,5 +1,8 @@
 package rofl
 
+// LatestBuilderImage is the latest builder container image to use when building ROFL apps.
+const LatestBuilderImage = "ghcr.io/oasisprotocol/rofl-dev:v0.5.0@sha256:31573686552abeb0edebc450f6872831f0006a6cf38220cef7e0789d4376c2c1"
```

Comment:
Actually, this is a bit awkward, because `rofl-dev` contains CLI. So we need to release a new CLI version to bump it in the rofl-dev, but then we need to update the ref in CLI. Hm but do we even need CLI there?

Reply:
`rofl-dev` is also for container-everything cases where people don't want to install anything on their PC: https://docs.oasis.io/build/rofl/workflow/prerequisites#conservative-containers-everywhere

Reply:
Yes, we should use a different builder and build it in this repository.